### PR TITLE
Tweak the Navigator class doc

### PR DESCRIPTION
**Pull request by HansMuller**

### CODE REVIEW

**Comment by abarth** [comment]
LGTM

File: packages/flutter/lib/src/widgets/navigator.dart [hunk 1]
```diff
@@ -234,9 +234,12 @@ typedef bool RoutePredicate(Route<dynamic> route);
 /// Many apps have a navigator near the top of their widget hierarchy in order
 /// to display their logical history using an [Overlay] with the most recently
 /// visited pages visually on top of the older pages. Using this pattern lets
-/// the navigator visually transition from one page to another by the widgets
+/// the navigator visually transition from one page to another by moving the widgets
 /// around in the overlay. Similarly, the navigator can be used to show a dialog
 /// by positioning the dialog widget above the current page.
+///
+/// Although you can create a navigator directly, it's most common to refer to the
+/// navigator created by a [WidgetsApp] or a [MaterialApp] with [Navigator.of].
```
Comment (Hixie) [inline packages/flutter/lib/src/widgets/navigator.dart]: s/refer to/use/
s/with/ /widget. You can refer to that [Navigator] using [Navigator.of]./

**Comment by Hixie** [comment]
LGTM

File: packages/flutter/lib/src/widgets/navigator.dart (hunk 1 above)
Comment (HansMuller) [inline packages/flutter/lib/src/widgets/navigator.dart]: Much better.

**Pull request merged**